sklearn/decomposition/tests/test_pca.py

```diff
@@ -471,6 +471,26 @@ def test_infer_dim_by_explained_variance():
     assert_equal(pca.n_components_, 2)
 
 
+def test_pca_score_equal():
+    # ensure that the scores calculated by different svd_solver are equal
```

Comment:
Isn't this already tested by `test_pca_score`? can we now test equality there at a higher number of decimal places?